tools-local/tasks/mobile.tasks/AppleAppBuilder/Templates/main-console.m

```diff
@@ -31,15 +31,17 @@ @implementation ViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
 
-    CGRect applicationFrame = [[UIScreen mainScreen] applicationFrame];
+    CGRect applicationFrame = [[UIScreen mainScreen] bounds];
     logLabel = [[UITextView alloc] initWithFrame:
         CGRectMake(2.0, 50.0, applicationFrame.size.width - 2.0, applicationFrame.size.height - 50.0)];
     logLabel.font = [UIFont systemFontOfSize:9.0];
     logLabel.backgroundColor = [UIColor blackColor];
     logLabel.textColor = [UIColor greenColor];
     logLabel.scrollEnabled = YES;
     logLabel.alwaysBounceVertical = YES;
+#ifndef TARGET_OS_TV
```

Comment:
please also update runtime.m since we have a bunch of TARGET_OS_IPHONE checks there that need to be updated with tvOS too.